PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/mysql/MySQLPropertyView.java

```diff
@@ -225,7 +226,7 @@ public void onReadProperty(String sid, String resourceGroup, String name) {
             this.showProperty(this.property);
         };
         // show property in background
-        String taskTitle = String.format(actionName + StringUtils.SPACE + MySQLModule.ACTION_PATTERN_SUFFIX, name);
+        String taskTitle = Node.getProgressMessage(actionName, MySQLModule.MODULE_NAME, name);
```

Comment:
rename to `getTaskTitle`/`getActionTitle`?

Reply:
Progress is build-in concept in Intellij. maybe it should be more proper. shouldn't it ?